src/util/threadnames.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2018 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#ifndef BITCOIN_THREADUTIL_H
+#define BITCOIN_THREADUTIL_H
```

Comment:
In commit "threads: introduce threadutil, refactor thread naming" (1cf8779ee2887e3c8ea0265fe8bbb9ecbd502172)

Should use `BITCOIN_UTIL_THREADNAMES_H`. Also commit message text reference previous names and could be updated.